common/src/Assets/Texture.h

```diff
@@ -33,8 +33,10 @@ namespace TrenchBroom {
         class TextureCollection;
         
         typedef Buffer<unsigned char> TextureBuffer;
-        void setMipBufferSize(TextureBuffer::List& buffers, const size_t width, const size_t height);
-        
+
+        size_t bytesPerPixelForFormat(GLenum format);
+        void setMipBufferSize(TextureBuffer::List& buffers, size_t width, size_t height, GLenum format);
```

Comment:
const, const, const ;-)

Reply:
For width/height/format, there's no need to make them `const` in the function declaration, right? CLion gives a "clang tidy warning" if you use const on these.

Reply:
oh right, sorry. By value params should not be const in the declaration, but in the definitions.